services/core-web/src/components/admin/contacts/EMLIContacts/EMLIContactsTable.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import { Button, Popconfirm } from "antd";
+import PropTypes from "prop-types";
+import { nullableStringSorter } from "@common/utils/helpers";
+import * as Strings from "@common/constants/strings";
+import { TRASHCAN, EDIT_OUTLINE_VIOLET } from "@/constants/assets";
+import CoreTable from "@/components/common/CoreTable";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
+import { CoreTooltip } from "@/components/common/CoreTooltip";
+
+const propTypes = {
+  isLoaded: PropTypes.bool.isRequired,
+  isOffice: PropTypes.bool,
+  openEditModal: PropTypes.func.isRequired,
+  handleDeleteContact: PropTypes.func.isRequired,
+  mineRegionHash: PropTypes.objectOf(PropTypes.string).isRequired,
+  EMLIContactTypesHash: PropTypes.objectOf(PropTypes.string).isRequired,
+  contacts: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+};
+
+const defaultProps = { isOffice: false };
+
+const hideColumn = (condition) => (condition ? "column-hide" : "");
+const columns = (
+  regionHash,
+  openEditModal,
+  handleDeleteContact,
+  isOffice,
+  EMLIContactTypesHash
+) => [
+  {
+    title: "Region",
+    dataIndex: "mine_region_code",
+    sorter: nullableStringSorter("mine_region_code"),
+    render: (text) => <div title="Region">{regionHash[text] || Strings.EMPTY_FIELD}</div>,
+  },
+  {
+    title: "Major or Regional Contact",
+    dataIndex: "is_major_mine",
+    sorter: (a, b) => (a.is_major_mine > b.is_major_mine ? -1 : 1),
+    render: (text) => (
+      <div title="Major or Regional Contact">{text ? "Major Mine" : "Reginal Mine"}</div>
```

Comment:
As mentioned in a general comment, small typo here.

Reply:
ah thank you. Fixed spelling